index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Project Day 2</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+    <header class="header">
+      <h3 class="logo">SJA</h3>
+      <nav class="nav">
+      <img src="img/Union.png" alt="Union"/>
+		</nav>
+	</header>
```

Comment:
Please wrap the logo in a link tag.
Please use font-family for the header section from the design page.
Please align the logo and hamburger menu vertically. Kindly see the screenshot below for reference.
![header](https://user-images.githubusercontent.com/35504149/133456929-41b2e9f7-a89d-4457-a15d-42752d40465f.png)